scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -2,82 +2,12 @@
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class BuildVersion
+    public class BuildVersion : Version
     {
-        public int Major { get; set; }
-        public int Minor { get; set; }
-        public int Patch { get; set; }
-        public int CommitCount { get; set; }
-        public string CommitCountString => CommitCount.ToString("000000");
-        public string ReleaseSuffix { get; set; }
-
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
-        public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
+        public string VersionSuffix => $"{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
`VersionSuffix` is used in multiple places including packing our test packages https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/TestTargets.cs#L204. So for test packages the nuget version will look like `1.0.0-002668`.

Also now the `NuGetVersion` which used everywhere in all the installers will now be just `1.0.0-002668` instead of `1.0.0-preview1-002668`